parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -67,4 +67,4 @@ SGX_NODE={{GetVariable "sgxNode"}}
 AUDITD_ENABLED={{GetVariable "auditdEnabled"}} 
 CONFIG_GPU_DRIVER_IF_NEEDED={{GetVariable "configGPUDriverIfNeeded"}}
 ENABLE_GPU_DEVICE_PLUGIN_IF_NEEDED={{GetVariable "enableGPUDevicePluginIfNeeded"}}
-/usr/bin/nohup /bin/bash -c "/bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1; systemctl --no-pager -l status kubelet 2>&1 | head -n 100"
\ No newline at end of file
+/usr/bin/nohup /bin/bash -c "/bin/bash /opt/azure/containers/provision.sh >> /var/log/azure/cluster-provision.log 2>&1 || PROVISION_CODE=$?; systemctl --no-pager -l status kubelet 2>&1 | head -n 100; exit ${PROVISION_CODE}"
```

Comment:
&& PROVISION_CODE=$?

we always want to get the code

Reply:
updated